server/src/DBTablesMonitoring.cc

```diff
@@ -998,6 +999,25 @@ string EventsQueryOption::getCondition(void) const
 		statusCondition = string("(") + statusCondition + string(")");
 	addCondition(condition, statusCondition);
 
+	string incidentStatusCondition;
+	for (auto status: m_impl->incidentStatuses) {
```

Comment:
And using 'const' is more suitable in this situation to avoid developers from changing its content by mistake.

const auto &status.